### PR TITLE
feat: improve anotation message

### PR DESCRIPTION
**Pull request by satoren**
recreate #96. I couldn't reopen it because I rebased it.

[annotation example](https://github.com/satoren/code-coverage-action/pull/2/files#diff-2b9858a6991811943ddc1371a0de750ab38276168ffa8c1c82961d59666e30f8R28)

I need help with the two comment places.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -10536,7 +10534,7 @@ const createGithubAccessToken = async (githubAppToken) => {
 
   const response = await makeRequest(query, variables);
 
-  if (!response.data && !response.data.createGithubAccessToken.success) {
+  if (!response.data || !response.data.createGithubAccessToken.success) {
```
Comment (satoren) [inline dist/index.js:10537]: When I execute `yarn build`, it will contain things that are not my changes.

File: src/features/showAnnotations.js [hunk 2]
```diff
@@ -22,20 +22,18 @@ const showAnnotations = async (compareFileData) => {
...
-        Array.isArray(line) ? line.join("-") : line
-      );
-
-      const linesWord = linesToDisplay.length === 1 ? "line is" : "lines are";
-
-      const formattedLines = linesToDisplay.join(", ");
-
-      const message = `file=${file}::${formattedLines} ${linesWord} not covered with tests`;
-
-      // NOTE: consider an option to show lines directly by attaching 'line' param
```
Comment (satoren) [inline src/features/showAnnotations.js:35]: I don't understand the case of this comment. Under what conditions does it occur?

File: dist/index.js (hunk 1 above)
Comment (vitaliimelnychuk) [reply dist/index.js:10537]: let me try to rebuild the master, maybe it was built not with newest version

Comment (vitaliimelnychuk) [reply dist/index.js:10537]: everything should be fine now, you should merge with newest master

File: src/features/showAnnotations.js (hunk 2 above)
Comment (vitaliimelnychuk) [reply src/features/showAnnotations.js:35]: I think this is more an issue for Node.js applications coverage reports, mentioned here https://github.com/barecheck/code-coverage-action/issues/40

I think it's great to remove this comment, I will check one more if we can get it fixed before running this action.

Comment (satoren) [reply src/features/showAnnotations.js:35]: OK. I'll revert back to comment.

**Review by vitaliimelnychuk — APPROVED** [review]

**Comment by vitaliimelnychuk** [comment]
Looks good, thanks for your contribution!

**Pull request merged**